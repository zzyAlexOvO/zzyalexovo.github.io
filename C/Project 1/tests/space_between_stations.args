"Brisbane Roma Street" Cardiff "10:59:22"