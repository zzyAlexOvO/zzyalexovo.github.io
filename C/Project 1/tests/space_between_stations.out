The next train to Cardiff from Brisbane Roma Street departs at 10:59:23
